src/assets/simp/build/simp.spec

```diff
@@ -163,7 +163,6 @@ Requires: pupmod-simp-simp_gitlab >= 0.6.0
 Requires: pupmod-simp-simp_grub >= 0.2.1
 Requires: pupmod-simp-simp_ipa >= 0.1.0
 Requires: pupmod-simp-simp_nfs >= 1.0.1
-Requires: pupmod-simp-simp_pki_service >= 0.3.1
```

Comment:
shouldn't there be an obsoletes by simp-extras?

Reply:
No, it's not yet obsoleted and does no harm if left on the system after 6.4

Reply:
The local Git repo will still exist if the RPM is uninstalled, so it still will be on the system.  My concern is that if we don't obsolete it here, we will forget and the RPM will hang around.

Reply:
It's OK if the RPM hangs around. Obsoletes is generally for when you want to make sure something gets ripped off the system because something else is replacing it.

In this case, there isn't any harm in having that on the system as well as the other modules (mixing them might cause issues but that's pretty normal testing type stuff).

This is part of the reason that everything is named `pupmod-<stuff>`. Users can easily find stragglers.